testing/trino-product-tests/src/main/java/io/trino/tests/product/mysql/TestJdbcDynamicFilteringJmx.java

```diff
@@ -50,9 +50,9 @@ public void testDynamicFilteringStats()
         assertThat(onTrino().executeQuery(format("SELECT COUNT(*) FROM mysql.%s a JOIN tpch.tiny.nation b ON a.nationkey = b.nationkey AND b.name = 'INDIA'", TABLE_NAME)))
                 .containsOnly(row(1));
 
-        assertThat(onTrino().executeQuery("SELECT \"completeddynamicfilters.totalcount\" FROM jmx.current.\"io.trino.plugin.jdbc:name=mysql,type=dynamicfilteringstats\""))
+        assertThat(onTrino().executeQuery("SELECT \"completeddynamicfilters.totalcount\" FROM jmx.current.\"io.trino.plugin.jdbc:name=mysql,type=dynamicfilteringstats\" WHERE node = 'presto-master'"))
```

Comment:
@raunaqmorarka - Since we have moved mysql to a multinode setup - we get the JMX metrics for worker also. Do we need this assertion only on the master or ?

Reply:
Checking these stats on coordinator should be sufficient